modules/runner/src/integrationTest/java/org/apache/ignite/internal/rest/ItInitializedClusterRestTest.java

```diff
@@ -38,6 +39,10 @@
  * Test for the REST endpoints in case cluster is initialized.
  */
 public class ItInitializedClusterRestTest extends AbstractRestTestBase {
+
+    private static final String IGNITE_SEMVER_REGEX =
```

Comment:
What does `SEMVER` mean?

Reply:
https://semver.org

Reply:
Do you think the naming might be more clear?

Reply:
Added the link to the filed comment